pkg/notes/notes_gatherer_test.go

```diff
@@ -494,6 +494,7 @@ func checkCallCount(t *testing.T, what string, expected, actual int) {
 	}
 }
 
+// nolint: unparam
```

Comment:
```
../../../../../../../../../../../../../../../../../../home/prow/go/src/k8s.io/release/pkg/notes/notes_gatherer_test.go:497:33: `checkOrgRepo` - `expectedOrg` always receives `"kubernetes"` (unparam)
func checkOrgRepo(t *testing.T, expectedOrg, expectedRepo, actualOrg, actualRepo string) {
```

We probably want `unparam` to do the check here.
Can you add a TODO to fix this or just address the unparam here?

Reply:
We can take this as a follow-up: https://kubernetes.slack.com/archives/CJH2GBF7Y/p1578671904030000?thread_ts=1578666872.026400&cid=CJH2GBF7Y